content/blog/2019-05-22-ramlegacy.md

```diff
@@ -0,0 +1,51 @@
+
+After ensuring that `ramlegacy` had a stable data source that was not constantly moving we set out to optimize the process of reading in the data after it’s been downloaded and cached on user’s computer. `load_ramlegacy` is the package function that allows the user to load table(s) from any specified version of the database. In the beginning in order to make the loading efficient we designed `load_ramlegacy` so that calling for a specified version would use `delayed_assign` to make all the dataframes within that version available in the user’s global environment. Additionally, we edited zzz.R to overload `library(ramlegacy)` so that it not only loaded the package but also called `load_ramlegacy` directly to load all the tables of the desired version. Later, our discussion with the reviewers helped us realize that overloading the `library` call would constrain the future development of the package as future may entail functionality outside the database. Plus this loading behaviour was outside what a user would expect from a typical `library` call. Finally we thought it best to give the users the ultimate control over where and when the data actually gets loaded so now only `load_ramlegacy` does the loading of data by returning a list of requested tables instead of delayed assigning all of them to the user’s global environment.
+
+## Related Works
+
+Sean Anderson has a namesake package that downloads the Microsoft Access version and converts it to a local sqlite3 database. Similar to Sean Anderson’s package, Jamie Ashander’s R package seems to be an R interface for the Microsoft Access version of the RAM Database and provides a set of functions using RPostgreSQL to connect to the database. Unfortunately, their packages no longer work in their current state, as they still point to the Wordpress URLs which are now defunct and would need to be replaced with the Zenodo DOIs. This again highlights the need for a transparent and relible data source for a data access package especially when that data has to reside outside of the package as is the case with `ramlegacy`.
+
+## Conclusion
+
+`ramlegacy` was developed to improve access to the RAM Legacy Stock Assessment Database by downloading, caching and loading the database according to user’s requirements. Hopefully, this post has been helpful in understanding how packages rely on data sources being stable and why it is important for package developers to establish a direct dialogue with the data providers if the data distribution method is unreliable and opqaue. Finally, thanks to rOpenSci’s [Brooke Anderson](https://github.com/geanders), [Sam Albers](https://github.com/boshek), and [Jamie Afflerbach](https://github.com/jafflerbach) for a great review process! We really appreciated the time and effort you guys put into suggesting all the important changes to the package.
```

Comment:
```suggestion
`ramlegacy` was developed to improve access to the RAM Legacy Stock Assessment Database by downloading, caching and loading the database according to user’s requirements. Hopefully, this post has been helpful in understanding how packages rely on data sources being stable and why it is important for package developers to establish a direct dialogue with the data providers if the data distribution method is unreliable and opqaue. Finally, thanks to rOpenSci’s [Brooke Anderson](authors/brooke-anderson/), [Sam Albers](authors/sam-albers/), and [Jamie Afflerbach](https://github.com/jafflerbach) for a great [review process](https://github.com/ropensci/software-review/issues/264)! We really appreciated the time and effort they put into suggesting all the important changes to the package.
```

Suggested change that can be accepted as a commit:
you guys -> they
for links to reviewers with an rOpenSci author page, I added link to that
added link to the review thread